<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configEdit</class>
 <widget class="QMainWindow" name="configEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Configuration</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fileNameLabel">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>File name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="description">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Background Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="backgroundVolume">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Announcements</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QListWidget" name="announceList">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="editGUIAnnounce">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editEditorAnnounce">
          <property name="text">
           <string>Edit with
Editor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editVimAnnounce">
          <property name="text">
           <string>Edit with 
Vim</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileManagerAnnounceButton">
          <property name="text">
           <string>Open in
File Manager</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshAnnounceButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Background specification</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QListWidget" name="backgroundList">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="editGUIBackground">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editEditorBackground">
          <property name="text">
           <string>Edit with
Editor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editVimBackground">
          <property name="text">
           <string>Edit with
Vim</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileManagerBackgroundButton">
          <property name="text">
           <string>Open in
File Manager</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshBackgroundButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="configSave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>SaveAs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveAsButton</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>saveAsButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>728</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>735</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configSave</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>configSaveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>726</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>cancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>730</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileManagerBackgroundButton</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>fileManagerBackgroundButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editVimBackground</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>editVimBackgroundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEditorBackground</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>editEditorBackgroundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editGUIBackground</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>editGUIBackgroundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileManagerAnnounceButton</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>fileManagerAnnounceButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editVimAnnounce</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>editVimAnnounceClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEditorAnnounce</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>editEditorAnnounceClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editGUIAnnounce</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>editGUIAnnounceClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshAnnounceButton</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>refreshAnnounceClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshBackgroundButton</sender>
   <signal>clicked()</signal>
   <receiver>configEdit</receiver>
   <slot>refreshBackgroundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>677</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>669</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveAsButtonClicked()</slot>
  <slot>configSaveClicked()</slot>
  <slot>cancelClicked()</slot>
  <slot>fileManagerBackgroundButtonClicked()</slot>
  <slot>editVimBackgroundClicked()</slot>
  <slot>editEditorBackgroundClicked()</slot>
  <slot>editGUIBackgroundClicked()</slot>
  <slot>fileManagerAnnounceButtonClicked()</slot>
  <slot>editVimAnnounceClicked()</slot>
  <slot>editEditorAnnounceClicked()</slot>
  <slot>editGUIAnnounceClicked()</slot>
  <slot>refreshAnnounceClicked()</slot>
  <slot>refreshBackgroundClicked()</slot>
 </slots>
</ui>
